hydro_cli_examples/examples/topolotree/prims/many_locations.py

```diff
@@ -0,0 +1,62 @@
+import random
+from global_land_mask import globe
+from math import radians, cos, sin, asin, sqrt
+
```

Comment:
There is a lot of redundant code across the various py files for prims and kruskals. As a general principle, you don't want to have the same code copy-pasted in two different files. Instead, put it in one place and import that where you need to use it. I would refactor it into a python file containing the code for each of the MST algorithms, a python file containing all the graph helper functions, a test suite of tests to verify the MST algorithms, and a driver program that has the gcp_regions and latencies data in it and launches the html visualizations that you're generating for each algorithm.